### ISSUE TITLE
error TS2307: Cannot find module '@kalimahapps/vue-icons/bs' or its corresponding type 

### ISSUE THREAD

**Issue by BiliBiliFox** [kalimah-apps/vue-icons#4]
``
1.
> cnpm install @kalimahapps/vue-icons
√ Installed 1 packages
√ Linked 1 latest versions
√ Run 0 scripts
√ All packages installed (used 25ms(network 22ms), speed 0B/s, json 0(0B), tarball 0B, manifests cache hit 1, etag hit 0 / miss 0

2.
yarn add @kalimahapps/vue-icons
yarn add v1.22.19
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning Workspaces can only be enabled in private projects.
[4/4] Building fresh packages...
warning Workspaces can only be enabled in private projects.
success Saved 1 new dependency.
info Direct dependencies
└─ @kalimahapps/vue-icons@1.1.6
info All dependencies
└─ @kalimahapps/vue-icons@1.1.6
Done in 6.00s.

3.
<script setup lang="ts">
import { BsCalendar4Range } from '@kalimahapps/vue-icons/bs';
</script>

<template>
<BsCalendar4Range />
</template>

4.
> yarn build
src/views/AboutView.vue:4:34 - error TS2307: Cannot find module '@kalimahapps/vue-icons/bs' or its corresponding type declarations.

``
why?
how?

**Comment by abdul-alhasany** [comment]
I have not tested the package with yarn yet. Have you tried with npm or pnpm?

**Comment by femans** [comment]
I encountered the same thing. It has to do with using typescript; is there a type declaration for this? 
Anyway, @dawnlike, in abscence of this you can add:
```
declare module "@kalimahapps/vue-icons/*";
```
to e.g. `env.d.ts` or a new `.d.ts` file, which should be then included in your tsconfig somewhere.

**Comment by abdul-alhasany** [comment]
You are right. I realized few days after I answered the question. There is no type declaration at the moment. However, you can resolve by creating a `d.ts` file, for example `vue-icons.d.ts` and this content to it:

```ts
declare module '@kalimahapps/vue-icons/bs';

// Add more for each icon set that you use e.g.
// declare module '@kalimahapps/vue-icons/io';
// declare module '@kalimahapps/vue-icons/md';
// ...
```

**Comment by abdul-alhasany** [comment]
Fixed in 1.3 release.
